src/pages/MyKiva/MyKivaPageContent.vue

```diff
@@ -202,37 +175,139 @@ const selectedJourney = ref('');
 const showNavigation = ref(false);
 const showSideSheet = ref(false);
 const state = ref(STATE_JOURNEY);
-const totalUpdates = ref(0);
 const updatesLimit = ref(3);
-const updatesOffset = ref(0);
+const updatesOffset = ref(updatesLimit.value);
```

Comment:
The offset isn't working like expected now. When you click the load more button, it's supposed to load more and put the index at the first new card. Currently the first click of load more resets the index the first card.